jsonnet/kube-prometheus/blackbox-exporter/blackbox-exporter.libsonnet

```diff
@@ -34,15 +34,22 @@ local kubeRbacProxyContainer = import '../kube-rbac-proxy/container.libsonnet';
       modules: {
         http_2xx: {
           prober: 'http',
+          http: {
+            preferred_ip_protocol: 'ip4',
```

Comment:
Should we also specify `ip_protocol_fallback: true` to allow falling back to IPv6?

Reply:
nvm, `ip_protocol_fallback: true` is the default.
LGTM

Reply:
makes sense, yes